docs/src/user/install-from-galaxy.rst

```diff
@@ -47,5 +47,5 @@ you can use virtualenv, e.g.::
 
    python3 -m venv $HOME/os_migrate_venv
    source $HOME/os_migrate_venv/bin/activate
-   pip install --upgrade 'openstacksdk>=0.36'
-   pip install --upgrade 'ansible>=2.9.1,<2.10'
+   python3 -m pip install --upgrade 'openstacksdk>=0.36'
+   python3 -m pip install --upgrade 'ansible>=2.9.1,<2.10'
```

Comment:
Is there a reason behind this change? Is this something recommended or better in some way? (I don't mind the change if it still does the job, just would like to know the rationale.)

Reply:
yeah its something im starting to see more and more, when we use venvs, so when you use pip directly you dont know which interpreter you will be using in the otehr case you do. let me get you some ref I was reading before this change

Reply:
i.e. this https://snarky.ca/why-you-should-use-python-m-pip/

Reply:
:+1: Thanks :)